src/sagemaker/jumpstart/exceptions.py

```diff
@@ -0,0 +1,69 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module stores exceptions related to SageMaker JumpStart."""
+
+from typing import List, Optional
+
+
+class VulnerableJumpStartModelError(Exception):
+    """Exception raised for errors with vulnerable JumpStart models."""
+
+    def __init__(
+        self,
+        model_id: Optional[str] = None,
+        version: Optional[str] = None,
+        vulnerabilities: Optional[List[str]] = None,
+        inference: Optional[bool] = None,
```

Comment:
for extensibility, would it be possible to rename this argument to `scope` and pass an `enum` here instead of a `bool` ?

That way, we could support packaging for edge, or batch in the future for example.

You can raise a `NotImplementedError` if the scope isn't mapped yet in the SDK.